components/patina_smbios/README.md

```diff
@@ -46,6 +46,88 @@ commands.add_component(SmbiosProvider::new(3, 9));  // SMBIOS 3.9
 > creation fails during `entry_point()`, the component returns
 > `EfiError::Unsupported` instead of panicking.
 
+## Platform Integration
+
+To integrate the `patina_smbios` component into your platform, you must remove
+the existing SMBIOS drivers and replace them with the Patina SMBIOS component.
+
+### Step 1: Remove Existing SMBIOS Drivers
+
+Remove the following entries from your platform's DSC file (e.g.,
+`YourPlatform.dsc` or `YourPlatformCommon.dsc.inc`):

```

Comment:
I'm hesitant to say that they **must** remove their platform's SmbiosPlatformDxe driver. Realistically they could keep their platforms Smbios platform driver, and everything would still work because your Patina component produces the protocol.

The above scenario will only be realistic when other components are developed that rely on the SmBios Patina service, which is not the case right now. But in that scenario, the platform would replace the EDKII `SmBiosDxe` driver and register the Patina component(s) without necessarily also replacing their platforms EDKII platform SmBios driver.